app/src/main/java/org/oppia/android/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -340,6 +407,9 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
     } else {
       // For showing navigation drawer in AdministratorControlsActivity
       getFooterViewModel().isAdministratorControlsSelected.set(true)
+      binding.fragmentDrawerNavView.menu.forEach {
+        it.isChecked = false
```

Comment:
@anandwana001 I added this piece of code here to ensure that whenever isAdminControlsSelected is set to true, any other item can not be checked, but it still failed.

Reply:
Changing it to ```isCheckable = false``` worked. Now all the bugs relevant to the issue have been resolved.